cli/cmd/inject.go

```diff
@@ -479,3 +483,19 @@ func parsePortRanges(portRanges []*cfg.PortRange) string {
 
 	return strings.TrimSuffix(str, ",")
 }
+
+// overwriteRegistry replaces the registry-portion of the provided image with the provided registry.
+func overwriteRegistry(image, registry string) string {
+	if image == "" || registry == "" {
+		return image
+	}
+	updated := registry
+	if !strings.HasSuffix(updated, "/") {
+		updated += "/"
+	}
+	i := len(image) - 1
```

Comment:
Could you use `strings.LastIndex` here?

Reply:
Yeah...didn't notice that method.  I'll update...I have a merge conflict to contend with anyway. 😄